### PR TITLE
Add repository support / contribution docs

### PR DESCRIPTION
**Pull request by corinagum**
Fix https://github.com/Microsoft/BotFramework-WebChat/issues/1867

- Added a Support resource, which will pop up on the right when a user begins filing a new issue
- Bug issue template
- Feature Request issue template
- Question issue template
- Contributing file (Repo PR rules will be added on a later date)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+### [Github issues](https://github.com/Microsoft/botframework-webchat/issues) should be used for bugs and feature requests. See the Support section to get support related to Bot Framework and Web Chat.
```
Comment (compulim) [inline .github/ISSUE_TEMPLATE/BUG.md]:
```suggestion
### [GitHub issues](https://github.com/Microsoft/botframework-webchat/issues) should be used for bugs and feature requests. See the Support section to get support related to Bot Framework and Web Chat.
```

File: .github/ISSUE_TEMPLATE/BUG.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+## Version
+<!-- What version of Web Chat are you using? Are you using the CDN? NPM package? Or embedding Web Chat to your site via `<iframe>`? -->
```
Comment (compulim) [inline .github/ISSUE_TEMPLATE/BUG.md]: One way to find out the version (for v4+) is looking at `<meta>` tag.

File: .github/ISSUE_TEMPLATE/BUG.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Describe the bug
+<!-- Give a clear and concise description of what the bug is. -->
+<!-- Please be sure to add screencaps of the console errors in your browser, if there are any -->
```
Comment (compulim) [inline .github/ISSUE_TEMPLATE/BUG.md]:
```suggestion
<!-- Please be sure to add screenshots of the console errors in your browser, if there are any -->
```

File: .github/ISSUE_TEMPLATE/BUG.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+### [Github issues](https://github.com/Microsoft/botframework-webchat/issues) should be used for bugs and feature requests. See the Support section to get support related to Bot Framework and Web Chat.
+
+## Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
```
Comment (compulim) [inline .github/ISSUE_TEMPLATE/BUG.md]: Do you think we should give notes about personal-identifiable information should not be in screenshots?

File: .github/ISSUE_TEMPLATE/FEATURE-REQUEST.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+### Use the ['Enhancement' query](https://github.com/Microsoft/botframework-Webchat/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aenhancement) to search for the most popular feature requests.
+### The [Sample tag](https://github.com/Microsoft/botframework-Webchat/issues?q=is%3Aissue+is%3Aopen+label%3ASample) also lists examples and implementations that have already been requested.
+
+**Is your feature request related to a problem? Please describe.**
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+**Describe the suggestion or request in detail**
+<!-- A clear and concise description of what you want to happen. -->
+
+**Describe alternatives you've considered**
```
Comment (compulim) [inline .github/ISSUE_TEMPLATE/FEATURE-REQUEST.md]:
```suggestion
**Describe alternatives you have considered**
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Fixes #<!-- If this addresses a specific issue, please provide the issue number here -->
+
+- [ ] CHANGELOG.md updated
+
+## Description
+<!-- Please discuss the changes you have worked on. What do the changes do; why is this PR needed? -->
+
+## Specific Changes
```
Comment (compulim) [inline .github/PULL_REQUEST_TEMPLATE.md]: Could be replaced with "## Changelog" and devs can just copy-and-paste their changelog here, to lower their workload.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+<!-- Please list the changes in a concise manner. -->
+
+  -
+  -
+  -
+
+## Testing
+<!-- If you are adding a new feature to a library, you must include tests for your new code. -->
```
Comment (compulim) [inline .github/PULL_REQUEST_TEMPLATE.md]: This one looks like a checkbox than a section.

Or we can also turn this one into screenshots of your great work.

Your call.

File: .github/ISSUE_TEMPLATE/QUESTION.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (compulim) [inline .github/ISSUE_TEMPLATE/QUESTION.md]: I know we want to discourage people using GitHub issues as QA platform.

Should we turn this into FAQs too? Like, IE11 ES5, etc.

Comment (corinagum) [reply .github/ISSUE_TEMPLATE/QUESTION.md]: https://github.com/Microsoft/BotFramework-WebChat/issues/1961

**Pull request merged**